src/main/java/net/dv8tion/jda/api/utils/SessionControllerAdapter.java

```diff
@@ -104,8 +141,10 @@ public void handleResponse(Response response, Request<ShardedGateway> request)
 
                         String url = object.getString("url");
                         int shards = object.getInt("shards");
+                        int concurrency = object.getObject("session_start_limit")
+                                                .getInt("max_concurrency", 1);
```

Comment:
This is missing in master after release merge. Is that ok?

Reply:
So it means that actual `max_concurrency` is never set to SessionController now in master branch and always equals 1.

Reply:
Fixed in bdcece1